common/beerocks/bcl/include/bcl/beerocks_socket_thread.h

```diff
@@ -29,7 +31,7 @@ class socket_thread : public thread_base {
     socket_thread(const std::string &unix_socket_path_ = std::string());
     virtual ~socket_thread();
     void set_server_max_connections(int connections);
-    virtual void set_select_timeout(unsigned msec);
+    virtual void set_select_timeout(unsigned msec, bool sticky = true);
```

Comment:
if you are changing the signature then at least add doxygen that explains the new parameter

Reply:
I revert this change and reimplement it as @vitalybu has proposed.